uspace/app/bdsh/cmds/modules/alias/alias.c

```diff
@@ -130,25 +153,24 @@ int cmd_alias(char **argv)
 
 	size_t i;
 	for (i = 1; argv[i] != NULL; i++) {
-		char *name = str_dup(argv[i]);
+		char *name = argv[i];
```

Comment:
Nitpick: this variable can be `const char *` now.

Reply:
I think you did not see that I assign the simboly '\0' to it just two lines bellow

Reply:
Ah, I missed that.